js_modules/dagit/packages/core/src/workspace/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -36,11 +40,37 @@ export const SidebarAssetInfo: React.FC<{
       </SidebarSection>
 
       <div style={{borderBottom: `2px solid ${ColorsWIP.Gray300}`}} />
+      {inProgressRunIds?.length > 0 && (
+        <Box
+          background={ColorsWIP.Blue50}
+          flex={{direction: 'row', gap: 4, alignItems: 'center'}}
+          border={{side: 'bottom', width: 1, color: ColorsWIP.Blue100}}
+          padding={{vertical: 12, left: 24, right: 12}}
+          style={{color: ColorsWIP.Blue700, fontSize: 12, fontWeight: 700}}
+        >
+          <span>This asset may be materialized soon by </span>
+
+          {['12341514'].map((runId) => (
+            <Link to={`/instance/runs/${runId}`} key={runId}>
```

Comment:
I think elsewhere we put the link on the ID text or other inner info, rather than the tag itself.

Reply:
Ahh yeah I can move the link to the ID text 👍